src/tic_tac_toe/minimax.clj

```diff
@@ -0,0 +1,134 @@
+(ns tic-tac-toe.minimax
+  (:require [tic-tac-toe.board :as board]
+            [tic-tac-toe.rules :as rules]
+            [tic-tac-toe.player :as player]))
+
+(defn score [token]
+  (cond
+    (= token player/computer) 10
+    (= token player/human) -10
+    :else 0))
+
+(defn score-board [board]
+  (score (rules/winner board)))
+
+(defn available-spaces [board]
+  (for [x board
```

Comment:
I would rename x to something more descriptive maybe space, slot, index, something that describes that it is.